### ISSUE TITLE
Hi Bharat

### ISSUE THREAD

**Issue by ancoreraj** [bharat-1809/bharat-1809#1]
I was just thinking about what if we could send messages to our favourite developers on github.
Then this idea popped into my head.

Sorry for disturbing you.

**Comment by bharat-1809** [comment]
Yeah, it's a great idea. Like there can be some board/thread where we can praise the developers of the project, give them new ideas, and so on. But in my opinion direct messaging would be an overkill for GitHub. Since the devs are already busy solving issues and building stuff, they hardly will have time to interact with the message.

We can have some "wall" though, where users can give kudos and send suggestions. But I guess GitHub already uses the `discussions` tab for that.

And no bro, you aren't disturbing me 😅 .
